vendor/bat-native-ads/src/bat/ads/new_tab_page_ad_info.cc

```diff
@@ -46,156 +48,137 @@ bool NewTabPageAdInfo::IsValid() const {
+  if (const auto* value = root.FindString("creative_instance_id")) {
+    creative_instance_id = *value;
   }
 
-  if (document.HasMember("creative_instance_id")) {
-    creative_instance_id = document["creative_instance_id"].GetString();
+  if (const auto* value = root.FindString("creative_set_id")) {
+    creative_set_id = *value;
   }
 
-  if (document.HasMember("creative_set_id")) {
-    creative_set_id = document["creative_set_id"].GetString();
+  if (const auto* value = root.FindString("campaign_id")) {
+    campaign_id = *value;
   }
 
-  if (document.HasMember("campaign_id")) {
-    campaign_id = document["campaign_id"].GetString();
+  if (const auto* value = root.FindString("advertiser_id")) {
+    advertiser_id = *value;
   }
 
-  if (document.HasMember("advertiser_id")) {
-    advertiser_id = document["advertiser_id"].GetString();
+  if (const auto* value = root.FindString("segment")) {
+    segment = *value;
   }
 
-  if (document.HasMember("segment")) {
-    segment = document["segment"].GetString();
+  if (const auto* value = root.FindString("company_name")) {
+    company_name = *value;
   }
 
-  if (document.HasMember("company_name")) {
-    company_name = document["company_name"].GetString();
+  if (const auto* value = root.FindString("image_url")) {
+    image_url = GURL(*value);
   }
 
-  if (document.HasMember("image_url")) {
-    image_url = GURL(document["image_url"].GetString());
+  if (const auto* value = root.FindString("alt")) {
+    alt = *value;
   }
 
-  if (document.HasMember("alt")) {
-    alt = document["alt"].GetString();
+  if (const auto* value = root.FindString("target_url")) {
+    target_url = GURL(*value);
   }
 
   wallpapers.clear();
-  if (document.HasMember("wallpapers")) {
-    for (const auto& wallpaper : document["wallpapers"].GetArray()) {
-      NewTabPageAdWallpaperInfo wallpaper_info;
-      auto iterator = wallpaper.FindMember("image_url");
-      if (iterator == wallpaper.MemberEnd() || !iterator->value.IsString()) {
+  if (const auto* wallpapers_list = root.FindList("wallpapers")) {
+    for (const auto& item : *wallpapers_list) {
+      if (!item.is_dict())
         continue;
-      }
-      wallpaper_info.image_url = GURL(iterator->value.GetString());
 
-      iterator = wallpaper.FindMember("focal_point");
-      if (iterator == wallpaper.MemberEnd() || !iterator->value.IsObject()) {
+      const auto& wallpaper = item.GetDict();
+      const auto* image_url = wallpaper.FindString("image_url");
+      const auto* focal_point = wallpaper.FindDict("focal_point");
+      if (!image_url || !focal_point)
         continue;
-      }
 
-      auto focal_point = iterator->value.GetObject();
-      iterator = focal_point.FindMember("x");
-      if (iterator == focal_point.MemberEnd() || !iterator->value.IsInt()) {
+      const auto x = focal_point->FindInt("x");
+      const auto y = focal_point->FindInt("y");
+      if (!x || !y)
         continue;
```

Comment:
nit: for consistency in ads can we please wrap with `{ }`

Reply:
Done in e16dbf93d8230c489acbb7c2b905c7e20fc95930.